doc.go

```diff
@@ -62,6 +62,17 @@ const (
 	claimResourceAccess = "resource_access"
 	claimResourceRoles  = "roles"
 	claimGroups         = "groups"
+
+	accessCookie       = "kc-access"
+	refreshCookie      = "kc-state"
+	requestURICookie   = "request_uri"
+	requestStateCookie = "OAuth_Token_Request_State"
```

Comment:
@fredbi Even though I understand you would like to bring awareness. I'd suggest to just rename to `httpScheme` and `httpsScheme`.

Reply:
httpsScheme stutters...

Reply:
@fredbi ok, that's fair enough.